regression/smt2_solver/basic-bv1/weber_qf_bv.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
+weber_qf_bv.smt2
+
+^EXIT=0$
+^SIGNAL=0$
+^unsat$
+--
```

Comment:
Can you add a version of this with the problematic operators commented out?

Reply:
done

Reply:
Thanks.